Source/WebCore/style/AnchorPositionEvaluator.h

```diff
@@ -56,15 +56,30 @@ struct AnchorPositionedState {
     AnchorPositionResolutionStage stage;
 };
 
+// https://drafts.csswg.org/css-anchor-position-1/#typedef-anchor-size
+enum class AnchorSizeSize {
```

Comment:
AnchorSizeType maybe ? also we usually specify the type of the enum class

Reply:
Yeah I have a hard time naming this 😅 That could work I guess, I've also been thinking of naming it `AnchorSizeLength` 🤔

Reply:
other possible names are AnchorSizeDimension or AnchorSizeSource

Reply:
I decided on `AnchorSizeDimension`